rdt/transformers/base.py

```diff
@@ -20,11 +20,11 @@ class BaseTransformer:
     DETERMINISTIC_REVERSE = None
     COMPOSITION_IS_IDENTITY = None
     IS_GENERATOR = None
-    NEXT_TRANSFORMERS = None
 
     columns = None
     column_prefix = None
     output_columns = None
+    _next_transformers = None
```

Comment:
We can, but then we don't have the functionality you mentioned in the issue:
```python3
float_formatter._get_next_transformers()
{
    None: None,
    'is_null': None
}
```

Reply:
That's a good idea. My question then is whether we want to store the next transformers or not? Because if we use the method above, there's no point in creating the ``self._next_transformers`` attribute, we can get the next transformers by calling the function.